app/Http/Controllers/HomeController.php

```diff
@@ -25,4 +26,97 @@ public function index()
     {
         return view('admin.home');
     }
+
+    /**
+     * Show the user cart
+     *
+     * @return \Illuminate\Contracts\Support\Renderable
+     */
+    public function viewCart()
+    {
+        return view('client.cart.index');
+    }
+
+    /**
+     * Add Product to Cart.
+     *
+     * @param  int  $id
+     * @return \Illuminate\Http\Response
+     */
+    public function addProductToCart($id)
+    {    
+        $product = Product::findOrFail($id);        
+        $cart = session()->get('cart');
+
+        // if cart is empty then this the first product
+        if (!$cart) {
+            $cart = [
+                    $id => [
+                        "name" => $product->name,
+                        "quantity" => 1,
+                        "price" => $product->price,
+                        "featured_img" => $product->featured_img,                        
+                    ],
+            ];
```

Comment:
```suggestion
            $cart = [
                $id => [
                    "name" => $product->name,
                    "quantity" => 1,
                    "price" => $product->price,
                    "featured_img" => $product->featured_img,                        
                ],
            ];
```
indent 4 space nhé a